kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/batch/GCSBatchTableWriter.java

```diff
@@ -74,8 +74,10 @@ private GCSBatchTableWriter(List<RowToInsert> rows,
   public void run() {
     try {
       writer.writeRows(rows, tableId, bucketName, blobName);
-    } catch (ConnectException | InterruptedException ex) {
-      throw new ConnectException("Thread interrupted while batch writing to BigQuery.", ex);
+    } catch (ConnectException ex) {
+      throw new ConnectException("Failed to write rows to GCS", ex);
+    } catch (InterruptedException ex) {
+      throw new ConnectException("Thread interrupted while batch writing to GCS.", ex);
```

Comment:
Is it easy at this point to indicate the name of the connector in the log?  Or some identifying mark so we can know which connector is generating the error.

Reply:
hm, I'll look into it.

Reply:
@mtagle you might look to see if Kafka connect now includes connector info in the log context ([MDC](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/MDC.html)). If so, SREs just need to include a new pre-defined variable.

Yup. https://issues.apache.org/jira/browse/KAFKA-3816 .. I recommend just using this once we get to 2.3, since it looks like this is when it was added.

Reply:
so I can't give the connector name but I can give the name of the table it's attempting to get data into. That is hopefully helpful?